catroid/src/main/res/values/styles.xml

```diff
@@ -466,9 +466,48 @@
         <item name="android:background">@drawable/brick_control_2h</item>
     </style>
 
+    <style name="FormulaEditorComputeButton">
```

Comment:
Please make a parrent-style. https://stackoverflow.com/questions/18056211/how-can-i-use-a-style-within-another-style-for-a-button